ScienceJournal/Resources/Strings.bundle/en.lproj/Localizable.strings

```diff
@@ -1195,6 +1195,9 @@
 /* Long description of an unknown bluetooth sensor displayed below a representative image on the Learn More page [CHAR_LIMIT=NONE] */
 "sensor_desc_second_paragraph_unknown_bluetooth" = "Bluetooth is a wireless technology standard using radio waves to exchange data over short distances from fixed and mobile devices.";
 
+/* Message to display to the user when bluetooth is not on but a bluetooth sensor is in the experiment on Observe [CHAR_LIMIT=200] */
+"bluetooth_not_on_body" = "Cannot connect to external sensor: Bluetooth is disabled. Turn on Bluetooth to connect to your external sensor.";
```

Comment:
Nit: We tend to use `_message` in the key of strings that will be presented to the user in an alert or snackbar. Can you make the key `bluetooth_hardware_disabled_message`?

Reply:
yes sure will change that.